tauri-app/src/lib/components/tables/OrderVaultsVolTable.svelte

```diff
@@ -54,8 +54,8 @@
     <TableBodyCell tdClass="break-all py-2" data-testid="total-out">
       {formatUnits(BigInt(item.totalOut), Number(item.token.decimals ?? 0))}
     </TableBodyCell>
-    <TableBodyCell tdClass="break-all py-2" data-testid="total-vol">
-      {formatUnits(BigInt(item.totalVol), Number(item.token.decimals ?? 0))}
```

Comment:
i don't understand why we removed total volume? i was expecting to see both total and net